spec/models/edition_spec.rb

```diff
@@ -27,12 +27,13 @@
 
   describe ".create_initial" do
     let(:document) { build(:document) }
+    let(:document_type_id) { build(:document_type, path_prefix: "/prefix").id }
```

Comment:
Do we need `path_prefix: "/prefix"` specified for the document_type for these tests?

Reply:
yeah, stuff breaks without it